src/controller/python/chip/yaml/format_converter.py

```diff
@@ -157,7 +156,9 @@ def convert_to_data_model_type(field_value, field_type):
 
         for t in typing.get_args(field_type):
             # Comparison below explicitly not using 'isinstance' as that doesn't do what we want.
-            if t != Nullable and t != type(None):
+            # type_none is simple hack for Flake8 E721
+            type_none = type(None)
+            if t != Nullable and t != type_none:
```

Comment:
How about something like this?:
```suggestion
            if t != Nullable and not isinstance(None, t):
```